lib/api/src/backend/unsupported.rs

```diff
@@ -0,0 +1,83 @@
+#![allow(dead_code)]
+
+use std::marker::PhantomData;
```

Comment:
We chatted about not doing this strategy (as it can infinitely open up the API surface), but only exposing the sys API through the main lib as deprecated exported fields to prompt the user to use `::sys` instead